From 1756ce14d2361b5640c0a868ab7d6d5b71662724 Mon Sep 17 00:00:00 2001
From: Ng Khai Wen <khai.wen.ng@intel.com>
Date: Tue, 22 Jan 2019 18:02:57 +0800
Subject: [PATCH 1218/1334] media: intel-ipu4: [VIRT] Increase wait time

Multi virtual channel requires longer wait time
for all of the I2C write to the register to
powerup the sensors.

Change-Id: Iaf3370f6bfbd65c535ace961dff147e1ea556634
Signed-off-by: Ng Khai Wen <khai.wen.ng@intel.com>
---
 drivers/media/pci/intel/virtio/intel-ipu4-virtio-be-stream.c | 2 +-
 drivers/media/pci/intel/virtio/intel-ipu4-virtio-fe.c        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/virtio/intel-ipu4-virtio-be-stream.c b/drivers/media/pci/intel/virtio/intel-ipu4-virtio-be-stream.c
index 7958fd8..ab5626c 100644
--- a/drivers/media/pci/intel/virtio/intel-ipu4-virtio-be-stream.c
+++ b/drivers/media/pci/intel/virtio/intel-ipu4-virtio-be-stream.c
@@ -20,7 +20,7 @@
 #include "intel-ipu4-virtio-be.h"
 
 #define MAX_SIZE 6 // max 2^6
-#define POLL_WAIT 5000 //5s
+#define POLL_WAIT 20000 //20s
 
 #define dev_to_stream(dev) \
 	container_of(dev, struct ici_isys_stream, strm_dev)
diff --git a/drivers/media/pci/intel/virtio/intel-ipu4-virtio-fe.c b/drivers/media/pci/intel/virtio/intel-ipu4-virtio-fe.c
index 5a02b82..517b48d 100644
--- a/drivers/media/pci/intel/virtio/intel-ipu4-virtio-fe.c
+++ b/drivers/media/pci/intel/virtio/intel-ipu4-virtio-fe.c
@@ -15,7 +15,7 @@
 
 static DEFINE_IDA(index_ida);
 
-#define REQ_TIMEOUT 6000 //6s
+#define REQ_TIMEOUT 22000 //22s
 
 struct ipu4_virtio_uos {
 	struct virtqueue *vq[IPU_VIRTIO_QUEUE_MAX];
-- 
2.7.4

